crates/router/src/types/api.rs

```
@@ -520,7 +520,9 @@ impl ConnectorData {
                 enums::Connector::Rapyd => {
                     Ok(ConnectorEnum::Old(Box::new(connector::Rapyd::new())))
                 }
-                // enums::Connector::Recurly => Ok(ConnectorEnum::Old(Box::new(connector::Recurly))),
+                enums::Connector::Recurly => {
+                    Ok(ConnectorEnum::Old(Box::new(connector::Recurly::new())))
+                }
```

Can we throw an error for billing processors ? just like we are doing it for FRM, etc processor